packages/cli/internal/signatures.js

```diff
@@ -18,7 +18,7 @@ function getFullFunctionSignature(functionAbi, functionParameters) {
 
   // Collect return values
   const outputDescriptions = functionAbi.outputs.map(
-    (output) => `${output.type}${` ${output.name}` || ''}`
+    (output) => `${output.type}${output.name ? ` ${output.name}` : ''}`
```

Comment:
Fixes an extra whitespace that was being shown on function signatures, from this:
```
getUintValue() view returns (uint256 ) 0x55ec6354
```
to this:
```
getUintValue() view returns (uint256) 0x55ec6354
```